packages/ontario-design-system-component-library/src/components/ontario-footer/ontario-footer.tsx

```diff
@@ -234,6 +234,13 @@ export class OntarioFooter {
 			};
 		}
 
+		if (termsOfUseLink) {
+			links['termsOfUseLink'] = {
+				href: termsOfUseLink.href,
+				text: termsOfUseLink.text ?? translations.termsOfUse[language],
+			};
+		}
```

Comment:
Do you know why we're adding to this property via an index accessor?  I think this one, and the `contactLink` one should use the property directly.  eg.

```ts
interface Fur {
	length?: number;
	colour: string;
}
interface Animal {
	name: string;
	age: number;
	tail?: boolean;
	fur?: Fur | false | null | undefined;
}

const cat: Animal = {
	name: 'Mr. Whiskers',
	age: 5,
};

cat.tail = true;
cat.fur = { colour: 'brown' };

console.log(cat);
```

This outputs:

```
{
  "name": "Mr. Whiskers",
  "age": 5,
  "tail": true,
  "fur": {
    "colour": "brown"
  }
}
```

It looks like property accessors will work as long as the value is being set at the property level, in this case an object.  This transformation here would look like:

```suggestion
			links.termsOfUseLink = {
				href: termsOfUseLink.href,
				text: termsOfUseLink.text ?? translations.termsOfUse[language],
			};
		}
```

The same thing can be done to `contactLink` as well.